.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is only the basic for prettier. There is another task that will add corresponding hooks